code/datums/holocall.dm

```diff
@@ -220,6 +222,14 @@
 	QDEL_NULL(record)
 	return ..()
 
+/obj/item/disk/holodisk/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/disk/holodisk))
+		var/obj/item/disk/holodisk/holodiskOriginal = W
+		record = holodiskOriginal.record

```

Comment:
This should probably copy it not reference, cleaning the disk deletes the record.